### PR TITLE
Add .gitignore

### PR DESCRIPTION
**Pull request by cdrage**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,59 @@
+#
+# PROJECT SPECIFIC
+#
+
+# Ignore compiled Kompose files
+opencompositor
```
Comment (surajssd) [inline .gitignore]: spell

**Comment by surajssd** [comment]
Other than spell correction rest LGTM!

**Comment by surajssd** [comment]
Merge once you correct that!

**Comment by cdrage** [comment]
@surajssd done! :)

**Pull request merged**